traffic_portal/conf/configDev.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// this is the config that is consumed by /server.js on traffic portal startup (sudo service traffic_portal start)
+module.exports = {
+    timeout: '120s',
+    useSSL: true, // set to true if you plan to use https (self-signed or trusted certs).
+    port: 80, // set to http port
+    sslPort: 443, // set to https port
+    // if useSSL is true, generate ssl certs and provide the proper locations.
+    ssl: {
+        key:    '../localhost.key',
```

Comment:
this requires it outside the TP directory. do you think it should be `./localhost.key` instead to keep it contained to TP?

Reply:
Yes, I reuse that key in other parts of the CDN so that's where I have it but having it in the TP dir makes more sense.